static/js/tabpanel.js

```diff
@@ -69,5 +69,18 @@ function triggerNextTab(tab) {
   if (!nextTab) {
     nextTab = document.querySelectorAll('.p-hero-tab__item')[0];
   }
+  recordEvent('carousel', 'autoplay', nextTab.id);
```

Comment:
Could we use this function in the click event handler (above line 10) to trigger the GA event, instead of using `onclick` in the markup?
`recordEvent('carousel', 'click', target.id);`

Reply:
I wanted to keep a consistent method of pushing events via the markup. I thought that would be better then changing it on this site and being different on others.

Reply:
I've seen it both ways on other projects, but if this is the preferred method I'll bear it in mind in future :+1: